postgres/problems/problem_14.md

```
@@ -1,21 +1,22 @@
+* WITH 句を使用して CTE を定義すること
+* ORDER BY sales_year DESC, sales_month DESC と FETCH FIRST 3 ROWS ONLY を使用すること
 
-## 想定出力例
-
-| day        |
-|------------|
-| 2024-02-03 |
-| 2024-02-14 |
-| …          |
+## 想定出力例（先頭 5 行）
+ 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)